config/crd/bases/operator.kyma-project.io_kymas.yaml

```diff
@@ -64,6 +64,7 @@ spec:
                       description: |-
                         Channel is the desired channel of the Module. If this changes or is set, it will be used to resolve a new
                         ModuleTemplate based on the new resolved resources.
+                        The Version and Channel are mutually exclusive options.
```

Comment:
Why is the version `v1beta1` extended?

Reply:
because most v1beta1 fields are from v1beta2 if they are consistent.